src/main/java/mate/web/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,42 @@
+package mate.web.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static Set<String> allowedUrls;
+
+    @Override
+    public void init(FilterConfig filterConfig) {
+        allowedUrls = new HashSet<>();
+        allowedUrls.add("/drivers/add");
+        allowedUrls.add("/login");
+    }
```

Comment:
I could  be wrong, but something here seems to me not ok. Will it be better this way?
```suggestion
    private static Set<String> allowedUrls = new HashSet<>();

    @Override
    public void init(FilterConfig filterConfig) {
        allowedUrls.add("/drivers/add");
        allowedUrls.add("/login");
    }
```

Reply:
Если брать в пример инициализацию в статическом блоке, то инициализация объекта списка тоже будет в статическом блоке) давай я оставлю, как есть, посмотрим, что скажет София в пн
![image](https://user-images.githubusercontent.com/22300600/126868613-a6cd3012-55da-46fa-9bcc-2be944008f9c.png)

Reply:
all is fine) we can initialize list in the init method